src/Microsoft.ML.Data/Transforms/DropSlotsTransform.cs

```diff
@@ -216,6 +216,18 @@ public ColInfoEx(SlotDropper slotDropper, bool suppressed, ColumnType typeDst, i
 
         private readonly ColInfoEx[] _exes;
 
+        /// <summary>
+        /// Convenience constructor for public facing API.
+        /// </summary>
+        /// <param name="env">Host Environment.</param>
+        /// <param name="input">Input <see cref="IDataView"/>. This is the output from previous transform or loader.</param>
+        /// <param name="name">Name of the output column.</param>
+        /// <param name="source">Name of the input column.  If this is null '<paramref name="name"/>' will be used.</param>
+        public DropSlotsTransform(IHostEnvironment env, IDataView input, string name, string source = null)
```

Comment:
I'm not sure this is a transform that works well for conveniences. Certainly this specific constructor, which really does nothing, is not something we need right now. I'd rather not do drop slots for now.